### ISSUE TITLE
Support todo-comments plugin

### ISSUE THREAD

**Issue by flexagoon** [sainnhe/gruvbox-material#199]
Currently this theme doesn't have support for the [todo-comments.nvim](https://github.com/folke/todo-comments.nvim) plugin so it uses default colors (eg. regular blue etc) which don't fit in that well. You can configure the colors manually, but it would be great to have this plugin supported OOTB.

**Comment by antoineco** [comment]
Duplicate of https://github.com/sainnhe/gruvbox-material/issues/120

Last time we checked, the plugin didn't have a good way to customize highlight groups. If that's the case now we can revisit this issue.

**Comment by antoineco** [comment]
I just checked and the situation is still the same as it was; the plugin tries to use core Neovim highlight groups to set its colors, but this doesn't work because gruvbox-material uses an underlined style for those particular groups instead of a foreground color. Ref. https://github.com/folke/todo-comments.nvim/blob/833d8dd8b07eeda37/lua/todo-comments/config.lua#L59-L66

For the time being I suggest using the following configuration in Neovim:

```lua
require 'todo-comments'.setup {
  colors = {
    error   = { 'Red'    },
    warning = { 'Yellow' },
    info    = { 'Blue'   },
    hint    = { 'Green'  },
  }
}
```

Which results in:

![image](https://user-images.githubusercontent.com/3299086/233808155-f2b5da3b-67c9-44a4-be0f-f7d06098d80d.png)

When I find the time, I will send a PR to the author to try addressing the issue with underlined default groups. I already did this with lualine, and the authors accepted my PR.
